plugins/woocommerce/tests/e2e/specs/admin-homescreen/welcome-modal.test.js

```diff
@@ -0,0 +1,5 @@
+const {
```

Comment:
Could this be removed now, given you are calling it in the `complete-onboarding-wizard.test.js` now?

Reply:
Nice catch! I addressed this change in commit d6c216f